packages/core/src/stylable-meta.ts

```diff
@@ -57,21 +64,50 @@ export class StylableMeta {
         this.vars = [];
         this.cssVars = {};
         this.keyframes = [];
-        this.elements = {};
-        this.classes = {
-            [RESERVED_ROOT_NAME]: rootSymbol,
-        };
-        this.mappedSymbols = {
-            [RESERVED_ROOT_NAME]: rootSymbol,
-        };
         this.mappedKeyframes = {};
         this.customSelectors = {};
         this.urls = [];
         this.scopes = [];
-        this.simpleSelectors = {};
         setFieldForDeprecation(this, `mixins`, { objectType: `stylableMeta` });
         this.mixins = [];
         this.transformDiagnostics = null;
         this.transformedScopes = null;
     }
+    getSymbol(name: string) {
```

Comment:
This is great APIs but you rarely use them

Reply:
Not sure if rarely, they also used all around the monorep.

These are public APIs, they are tested (each in it's own feature spec), and are replacements for the deprecated `meta.mappedSymbols`, `meta.classes`, and `meta.elements` that are used out of Stylbale